### PR TITLE
Expose _response_object as a property on BaseAPIJSONObject

### PR DESCRIPTION
**Pull request by kelseymorris95**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @kelseymorris95 has signed the CLA. Thanks for the pull request!

File: boxsdk/object/base_api_json_object.py [hunk 1]
```diff
@@ -61,3 +61,8 @@ def __repr__(self):
     def _description(self):
         """Return a description of the object if one exists."""
         return ""
+
+    @property
+    def response_object(self):
+        """Return the JSON object representing the object returned from a Box API request."""
```
Comment (jmoldow) [inline boxsdk/object/base_api_json_object.py]: Document ``:rtype: `dict` ``.

Also, we might want to return a copy of the `dict`, that way if they make modifications to the dict it won't affect the smart object. I will think about which behavior is preferable..

File: boxsdk/object/base_api_json_object.py [hunk 2]
```diff
@@ -61,3 +62,12 @@ def __repr__(self):
...
         return ""
+
+    @property
+    def response_object(self):
+        """
+        Return the JSON object representing the object returned from a Box API request.
+        :rtype:
+            `dict`
+        """
+        return copy.deepcopy(self._response_object)
```
Comment (jmoldow) [inline boxsdk/object/base_api_json_object.py]: Good idea.

**Comment by jmoldow** [comment]
👍 waiting for tests

**Pull request merged**